docs/grid_model/network_subnetwork.md

```diff
@@ -180,6 +181,9 @@ The `VoltageRegulatorOn` attribute is required. It voltage regulation is enabled
 
 Target values for generators (`TargetP` and `TargetQ`) follow the generator sign convention: a positive value means an injection into the bus. Positive values for `TargetP` and `TargetQ` mean negative values at the flow observed at the generator `Terminal`, as `Terminal` flow always follows load sign convention. The following diagram shows the sign convention of these quantities with an example.
 
+The `isCondenser` value corresponds for instance to generators which can control voltage even if their targetP is equal to zero.
```

Comment:
@jeandemanged or under Pmin?

Reply:
in OLF we have another parameter to disable voltage control if outside Pmin/Pmax